spikingjelly/activation_based/ann2snn/recipes/sta_transformer.py

```
@@ -0,0 +1,967 @@
+            if module is self:
+                continue
+            reset = getattr(module, "_reset_sta_state", None)
+            if reset is not None:
+                reset()
+
+    @staticmethod
+    def _add_outputs(a: Any, b: Any) -> Any:
+        if torch.is_tensor(a) and torch.is_tensor(b):
+            return a + b
+        if isinstance(a, tuple) and isinstance(b, tuple):
+            return tuple(_STAConvertedModel._add_outputs(x, y) for x, y in zip(a, b))
```

Ruff (0.15.18)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)